uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/broker/BrokerMicrosoftAuthenticator.java

```diff
@@ -342,6 +342,11 @@ protected void performDeviceRegistrationHelper(@NonNull final String username,
         }
     }
 
+    public void shouldUseDeviceSettingsPage(final boolean shouldUseDeviceSettingsPage) {
```

Comment:
Are we running our tests on previous versions of authenticator today? The reason I added the prev/updated variants for Authenticator impl is because the UI changed for authenticator and we wanted the test cases to succeed on both previous and updated versions of Auth app. Now that this bug has been fixed for a long time.. I think we should just let it fail on previous versions instead of adding these checks again. LMK if we still want the test cases to run on prev versions.

Reply:
We're still using older versions of authenticator when running the update scenario tests